neuromaps/datasets/atlases.py

```diff
@@ -198,7 +213,14 @@ def fetch_fslr(density='32k', url=None, data_dir=None, verbose=1):
 
 
 def fetch_mni152(density='1mm', url=None, data_dir=None, verbose=1):
-    keys = ['T1w', 'T2w', 'PD', 'brainmask', 'CSF', 'GM', 'WM']
+    if density == '1mm' or density == '2mm':
+        keys = ['2009cAsym_T1w', '2009cAsym_T2w', '2009cAsym_PD',
+                '2009cAsym_brainmask', '2009cAsym_CSF', '2009cAsym_GM',
+                '2009cAsym_WM', '6Asym_T1w', '6Asym_brainmask']
+    elif density == '3mm':
+        keys = ['2009cAsym_T1w', '2009cAsym_T2w', '2009cAsym_PD',
+                '2009cAsym_brainmask', '2009cAsym_CSF', '2009cAsym_GM',
+                '2009cAsym_WM']
```

Comment:
You can reduce duplication a bit (in case these get modified in the future) with something like:

```suggestion
    keys = ['2009cAsym_T1w', '2009cAsym_T2w', '2009cAsym_PD',
            '2009cAsym_brainmask', '2009cAsym_CSF', '2009cAsym_GM',
            '2009cAsym_WM']
    if density in ('1mm', '2mm'):
        keys += ['6Asym_T1w', '6Asym_brainmask']
```